pkgs/development/tools/minipro/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, installShellFiles, which, fetchgit, git, pkg-config, libusb1 }:
```

Comment:
It doesn't look like you're using either `git` or `which`, and the package seems to compile fine without them.